src/scipp/io/hdf5.py

```diff
@@ -382,7 +382,7 @@ def read(group: h5.Group, override=None):
         contents = {}
         contents['name'] = group.attrs['name']
         contents['data'] = _VariableIO.read(group['data'])
-        for category in ['coords', 'masks', 'attrs']:
+        for category in ['coords', 'masks']:
```

Comment:
Should we consider supporting legacy files and read from the attrs group as unaligned coords? Updating all the files in our tests data would be a fair amount of work.

Reply:
We could. But that can also break tests because the data arrays are suddenly different.

Reply:
That is easier to fix than recreating a lot of files?